framework/assets/yii.validation.js

```diff
@@ -405,15 +389,8 @@ yii.validation = (function ($) {
 
     function validateFile(file, messages, options) {
         if (options.extensions && options.extensions.length > 0) {
-            var index, ext;
-
-            index = file.name.lastIndexOf('.');
```

Comment:
What's the point of declaring `index` as `undefined` initially if then immediately assign some value to it?